docs/kubernetes/index.rst

```diff
@@ -1,140 +1,162 @@
+|                              |                        |                                 | the BIG-IP system. [#iapp]_                                        |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| node                         | X                      |                                 | All supported Kubernetes resources                                 |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| partition                    |                        | X                               | The |kctlr| cannot create or destroy BIG-IP partitions.            |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| pool                         | X                      |                                 | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| pool member                  | X                      |                                 | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| self IP                      |                        | X                               | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| profiles [#prof]_                                                                                                                                            |
++---+--------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+|   | HTTP                     |                        | X                               | :ref:`Kubernetes Ingress Resources <kctlr-ingress-config>`         |
+|   |                          |                        |                                 | (L7) only                                                          |
++---+--------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+|   | TCP                      |                        | X                               | Applies to all supported Kubernetes resources                      |
```

Comment:
Only ConfigMaps will use TCP. Routes and Ingress are HTTP by definition.

Reply:
ConfigMap can also do UDP.

These profiles also only refer to the basic profiles, they don't refer to setting optimized or customized versions.